mcc/odin/odin/odin.go

```diff
@@ -13,6 +13,15 @@ import (
 // Duration specified time to wait for instance to be available.
 var Duration = time.Duration(5) * time.Second
 
+// TrimLast trims last character from a given string and returns the trimmed
+// one.
+func TrimLast(in string) string {
```

Comment:
Would be better not to have it exported as its functionality is completely unrelated to odin's.

Reply:
Ok, I guess that if this is needed somewhere else, we will move to a separate library, then. ;)

Reply:
@miguelbernadi in order to have this working I made it unexported, but moved to `odin_test` package.